Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -206,9 +206,9 @@ static Ref<ElementType> createContainedElement(HTMLElement& container, const Ato
     return element;
 }
 
-void HTMLAttachmentElement::ensureModernShadowTree(ShadowRoot& root)
+void HTMLAttachmentElement::ensureWideLayoutShadowTree(ShadowRoot& root)
```

Comment:
I think this will need to be done in a follow-up PR though, so we don't have a mix of names in the meantime.

Reply:
We can also do renaming in a separate PR before this one and land this with more straightforward names, which I think is preferable over a follow up.

Reply:
The wide-layout UI is still not the default, it must be enabled through a preference, so I think it makes more sense for now to keep the naming as-is. The few name changes in this patch ("modern" -> "wide layout") just makes the code more consistent within itself and with the preference.
In the future, when we fully transition to the new UI everywhere and we get rid of the legacy UI, then it will make sense to drop the "wide layout" mentions.

Reply:
It's confusing because the "Implementation" enum refers to "Legacy" vs. "WideLayout" which kind of implies the legacy one is going away.

Reply:
We actually don't know for sure that some apps won't want to keep it.
I could rename it to "VerticalLayout". 🤔